src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetArea,DrawFigure {
```

Comment:
Тобі не потрібно імплементувати інтерфейси якщо ти наслідуєшся від класу, в якому вони вже імплементовані
```suggestion
public class Circle extends Figure {
```